### PR TITLE
Fix check of form fields because it didn't check correctly

### PR DESCRIPTION
**Pull request by kunicmarko20**
I am targeting this branch, because this is a fix.

## Changelog

```markdown
### Fixed
- Checking the wrong method for form fields
```

### CODE REVIEW

**Comment by kunicmarko20** [comment]
@axzx @kamilkuzma can you check if this fixes the exception?

**Comment by axzx** [comment]
for me its OK

**Comment by jvandesande** [comment]
This fixes the exception for me too!

File: tests/Controller/CRUDControllerTest.php [hunk 1]
```diff
@@ -1444,9 +1444,17 @@ public function testEditActionRuntimeException()
             ->with($this->equalTo('edit'))
             ->will($this->returnValue(true));
 
+        $form = $this->getMockBuilder(Form::class)
+            ->disableOriginalConstructor()
+            ->getMock();
```
Comment (greg0ire) [inline tests/Controller/CRUDControllerTest.php]: Please use `createMock` instead

Comment (kunicmarko20) [reply tests/Controller/CRUDControllerTest.php]: fixed.

**Comment by flolivaud** [comment]
Can you create a tag for this fix  : 3.38.2 
Thanks.

**Comment by kunicmarko20** [comment]
Already requested, a new version will be released soon.

**Comment by kunicmarko20** [comment]
https://github.com/sonata-project/SonataAdminBundle/releases/tag/3.38.2

**Pull request merged**